appendix-basic65.tex

```diff
@@ -8460,18 +8465,17 @@ \subsection{STOP}
 \item [Format:] {\bf STOP}
 \item [Usage:] Stops the execution
                of the BASIC program.
-               A message tells the line number of the break.
-               The {\bf READY.} prompt
-               appears and the computer goes into direct mode
+               A message will be displayed showing the line number of the break.
+               The \screentext{READY.} prompt
+               appears and the computer goes into direct mode,
                waiting for keyboard input.
-               The program execution can be resumed with {\bf CONT}.
 
 \item [Remarks:]
                All variable definitions are still valid after {\bf STOP}.
-               They may be inspected or altered and the
+               They may be inspected or altered, and the
                program may be continued with the {\bf CONT}
-               statement. Every editing of the program source
-               makes continuation impossible, however.
+               statement. However, any editing of the program source
+               makes continuation impossible.
```

Comment:
'impossible' sounds a bit heavy here. Perhaps instead of `makes continuation impossible`, could it be `will disallow any further continuation.`

Reply:
i agree, i didnt like that either